app/controllers/works_controller.rb

```diff
@@ -96,4 +111,12 @@ def validate_work_types!
     flash[:error] = errors.join("\n")
     redirect_to dashboard_path
   end
+
+  def submit_type
```

Comment:
consider changing this to a boolean. e.g.: `deposit?`

Reply:
yup, makes sense